node/src/reactor.rs

```diff
@@ -347,10 +352,26 @@ where
         let end = self.clock.end();
         let delta = self.clock.delta(start, end);
 
-        if delta > DISPATCH_EVENT_THRESHOLD {
+        let threshold = env::var(DISPATCH_EVENT_THRESHOLD_ENV_VAR)
+            .ok()
+            .and_then(|threshold_str| match u64::from_str(&threshold_str) {
+                Ok(threshold) => Some(Duration::from_micros(threshold)),
+                Err(error) => {
+                    if self.event_count % 1000 == 0 {
+                        warn!(
+                            "can't parse env var {}={} as a u64: {}",
+                            DISPATCH_EVENT_THRESHOLD_ENV_VAR, threshold_str, error
```

Comment:
Can't we do that in a `const` somewhere else instead of logging the warning for every event? :grimacing:

Reply:
You mean as a one-time thing?  I guess I could have something using https://doc.rust-lang.org/std/sync/struct.Once.html if you prefer.  I thought having the warning displayed more than once though would be useful to the user who had set the env var incorrectly, but having emitted on every event seemed like overkill.

Reply:
I am imagining that person trying to set this env var is running a deliberate test in which case maybe it makes sense to even panic?